src/Components/CookieConsentContext.js

```diff
@@ -1,48 +1,39 @@
 import * as React from "react";
-import * as Scrivito from "scrivito";
 
-export const CookieConsentContext = React.createContext({});
+const CookieConsentContext = React.createContext({});
 
 export function CookieConsentProvider(props) {
-  const [cookieConsent, setCookieConsent] = React.useState(() =>
+  const [cookieConsentChoice, setCookieConsentChoice] = React.useState(() =>
     getCookieConsent()
   );
 
   function updateCookieConsent(consent) {
     localStorage.setItem("CookieConsent", consent);
 
-    setCookieConsent(consent);
+    setCookieConsentChoice(consent);
   }
 
   return (
     <CookieConsentContext.Provider
       value={{
-        cookieConsent,
-        acceptCookieConsent: () => updateCookieConsent("accepted"),
-        declineCookieConsent: () => updateCookieConsent("declined"),
+        cookieConsentChoice,
+        setCookieConsentChoice: updateCookieConsent,
       }}
     >
       {props.children}
     </CookieConsentContext.Provider>
   );
 }
 
-export function cookieConsentUrl() {
-  const root = Scrivito.Obj.root();
-
-  if (!root) {
-    return null;
-  }
-
-  const cookieConsentLink = root.get("cookieConsentLink");
-
-  if (!cookieConsentLink) {
-    return null;
-  }
+export function useCookieConsent() {
```

Comment:
Let's keep all the logic ( `cookieConsentChoice`, `acceptCookieConsent` and `declineCookieConsent`) inside of `CookieConsentProvider`:

```js
export function useCookieConsent() {
  return React.useContext(CookieConsentContext);
}